socialhome/streams/app/components/Stream.vue

```diff
@@ -1,35 +1,41 @@
 <template>
-    <div>
-        <div class="container-flex">
-            <div v-show="$store.getters.hasNewContent" class="new-content-container">
-                <b-link @click.prevent.stop="onNewContentClick" class="new-content-load-link">
-                    <b-badge pill variant="primary">
-                        {{ translations.newPostsAvailables }}
-                    </b-badge>
-                </b-link>
-            </div>
-            <div v-images-loaded.on.progress="onImageLoad" v-masonry v-bind="masonryOptions">
-                <div class="stamped">
-                    <component :is="stampedElement" />
-                </div>
-                <div class="grid-sizer"></div>
-                <div class="gutter-sizer"></div>
-                <stream-element
-                    class="grid-item"
-                    v-masonry-tile
-                    v-for="content in $store.getters.contentList"
-                    :content="content"
-                    :key="content.id"
-                />
+    <div :class="{ container: this.$store.state.stream.single }">
+        <div v-show="$store.getters.hasNewContent" class="new-content-container">
+            <b-link @click.prevent.stop="onNewContentClick" class="new-content-load-link">
+                <b-badge pill variant="primary">
+                    {{ translations.newPostsAvailables }}
+                </b-badge>
+            </b-link>
+        </div>
+        <div v-if="this.$store.state.stream.single">
+            <stream-element
+                class="grid-item grid-item-full"
+                :content="$store.state.content"
+                :key="$store.state.content.id"
```

Comment:
`:key` is useless here has you do not iterate on a collection. (See [`key`](https://vuejs.org/v2/guide/list.html#key))